addons/website_sale_delivery/models/sale_order.py

```diff
@@ -18,11 +18,11 @@ class SaleOrder(models.Model):
     has_delivery = fields.Boolean(
         compute='_compute_has_delivery', string='Has delivery',
         help="Has an order line set for delivery", store=True)
-    website_order_line = fields.One2many(
-        'sale.order.line', 'order_id',
-        string='Order Lines displayed on Website', readonly=True,
-        domain=[('is_delivery', '=', False)],
-        help='Order Lines to be displayed on the website. They should not be used for computation purpose.')
+
+    @api.one
```

Comment:
`api.one` is deprecated, you can use `self.ensure_one()` with `api.multi` instead.